java/src/jmri/jmrit/display/layoutEditor/ConnectivityUtil.java

```diff
@@ -2726,8 +2726,8 @@ private String connectionTypeToString(int conType) {
             "SLIP_LEFT", "SLIP_RIGHT"};
         if (conType < con_types.length) {
             result = con_types[conType];
-        } else if ((LayoutTrack.BEZIER_CONTROL_POINT_OFFSET_MIN <= conType)
-                && (conType <= LayoutTrack.BEZIER_CONTROL_POINT_OFFSET_MAX)) {
+        } else 
+            if (LayoutTrack.isBezierHitType(conType)) {
```

Comment:
There seems to be a newline too much here since `else` and `if` are on different rows.

Reply:
Thx. I’ll fix that tomorrow; Travis-ci kicked up a few also.